services/userService.js

```diff
@@ -0,0 +1,110 @@
+const jwt = require('jsonwebtoken');
+const requestPromise = require('request-promise');
+const userDao = require('../models/userDao');
+
+const signIn = async ( code ) => {
+  const apiKey  = process.env.API_KEY;
+
+  const getKakaoTokenOption = {
+    uri: "https://kauth.kakao.com/oauth/token",
+    method: "POST",
+    form: {
+      grant_type: "authorization_code",
+      client_id: apiKey,
+      redirect_uri: "http://127.0.0.1:3000/user/kakao/signin",
+      code: code
+    },
+    headers: {
+        "content-type" : "application/x-www-form-urlencoded"
+    },
+    json: true
+  }
+
+  const getKakaoToken = await requestPromise(getKakaoTokenOption);
+
+  const getKakaoUserInfo = {
+    uri: "https://kapi.kakao.com/v2/user/me",
+    method: "POST",
+    headers: {
+        "content-type" : "application/x-www-form-urlencoded",
+        "Authorization" : `Bearer ${getKakaoToken["access_token"]}`
```

Comment:
변수를 새로 할당한다면 더 깔끔한 코드를 작성할 수 있을 것 같네요! 
```suggestion
    const KAKAO_TOKEN = getKakaoToken["access_token"]
   ....
        "Authorization" : `Bearer ${KAKAO_TOKEN}`
```